price-feeder/README.md

```diff
@@ -0,0 +1,144 @@
+# Oracle Price Feeder
+
+The `price-feeder` tool is an extension of Juno's `x/oracle` module, both of
+which are based on Terra's [x/oracle](https://github.com/terra-money/classic-core/tree/main/x/oracle) and Umee [x/oracle](https://github.com/umee-network/tree/umee/x/oracle) module.
+core differences are as follows:
+
+- All exchange rates must be quoted in USD or USD stablecoins.
+- No need or use of reference exchange rates (e.g. Luna).
+- No need or use of Tobin tax.
+- The `price-feeder` combines both `feeder` and `price-server` into a single
```

Comment:
All of these are differences from the terra fork to the umee fork, everything else here looks the same

Reply:
We'll add Juno swap as price provider. Yep it's implemented from Umee.

Reply:
ty for clarifying in docs @vuong177 💯